tools/depends/native/python3/Makefile

```diff
@@ -1,14 +1,12 @@
 include ../../Makefile.include
 PLATFORM=$(NATIVEPLATFORM)
-DEPS= ../../Makefile.include Makefile
+DEPS= ../../Makefile.include Makefile distutil.patch
 
 # lib name, version
 LIBNAME=Python
-VERSION=3.8.5
+VERSION=3.8.8
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.xz
-CWD=$(shell pwd)
-HOSTPYTHONDIR=$(CWD)/$(PLATFORM)/hostpython
 BASE_URL=https://www.python.org/ftp/python/$(VERSION)
```

Comment:
Python-3.8.8.tar.xz uploaded to mirrors.
https://mirrors.kodi.tv/build-deps/sources/Python-3.8.8.tar.xz